docs/plugins.md

```diff
@@ -774,6 +774,15 @@ The following plugins change the way that tile layers are loaded into the map.
 			<a href="https://github.com/allartk">Allart Kooiman</a>
 		</td>
 	</tr>
+	<tr>
+		<td>
+			<a href="https://github.com/z632896862/Leaflet.TileCorrection">Leaflet.TileCorrection</a>
+		</td><td>
+			Allow tiles to be loaded in an diffent crs from map's and start at a custom zoom.
```

Comment:
```suggestion
			Allow tiles to be loaded in an different crs from map's and start at a custom zoom.
```

English text at homepage also has typo:
> Waring：the projection of map's CRS should be same as the custom CRS

^^^^ Warning

Reply:
@z632896862 can you please update the text